bin/sbt

```diff
@@ -7,6 +7,6 @@ docker::set_project_name_dev
 
 bin/pull-image
 
-docker::run_shell_container
-docker::run_shell_command sbt "$@"
-docker::stop_shell_container
+docker::compose_dev up --wait -d
+docker exec -it "${COMPOSE_PROJECT_NAME}-civiform-1" ./entrypoint.sh -jvm-debug "*:8457" -Dsbt.offline
```

Comment:
suggestion (non-blocking): consider pulling `docker exec -it "${COMPOSE_PROJECT_NAME}-civiform-1" ./entrypoint.sh -jvm-debug "*:8457" -Dsbt.offline` into a bash function `docker::dev_server_sbt_command`

Reply:
Thanks, it's indeed cleaner that way. Done.